tasks/Restaurant.tex

```diff
@@ -71,16 +71,18 @@ \subsection*{Additional rules and remarks}
 		\item The robot can request to be guided to a customer's table.
 		\item The robot can choose to take several orders and place them later on, place an order and pick the next one while the former is being served, or dispatch one order at a time.
 		\item The robot can either transport each object individually, or using a tray. All delivered objects must be placed on the customer's table.
-		\item By default, the barman will place the order in a basket or tray for the robot to deliver it.
+		\item For transport with an unattached tray, the robot must pick up the objects and place them on a tray, pick up the tray, and then place the objects from the tray on the table (first placing the tray on the table is allowed).
+		\item If requested, the barman will place the order in a basket or tray for the robot to deliver it.
 		\item Upon arrival to the restaurant, only the team leader is allowed next to the robot for watching and charging.
 		\item If a person from the audience (severely) interferes with the robot in a way that makes it impossible to solve the task, the teams may repeat the test immediately.
 		\item Each Deus Ex Machina penalty for skipping manipulation will only be applied twice per order so receiving an order with three objects is not more punishing.
 		\item If the robot detects a customer but does not reach their table, the robot must clearly show who was detected to receive points, i.e. displaying a picture of the person.
+		\item When a team is at the front of the queue, they are allowed to start their mapping procedure (the robot must remain in place). When it is their turn, they must bring the robot directly and in a straight line from the front of the queue to the start location. Once at the start location only slight movements are allowed (no moving back and forth, no full rotations etc.)
```

Comment:
What if we say startup procedure or something?
This might mislead the team to create a map of the arena when they are waiting in the queue.

Reply:
Preparing in advance and starting the robot at the front of the queue is a good idea (no 30-second preparation is necessary anymore if they can prepare in advance).

> because I want to save as much time as possible at the start

Some teams might need to adjust the camera, mic, and software so we do not need to limit the mapping I think.